truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -114,6 +114,9 @@ def list_bucket_files(bucket_name, data_dir, is_trusted=False):
 
     all_objects = []
     for blob in blobs:
+        # leave out folders
```

Comment:
The folders will be included anyways. If you have `folder_a/` with an item `folder_a/image.png`, you can just copy over `folder_a/image.png` instead of copying both.

Reply:
I think a comment at the top of this function giving an example of the output would be helpful to help the reader understand why leaving out folders is safe